[2023-06-01T11:21:23.350+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-03-17T00:00:00+00:00 [queued]>
[2023-06-01T11:21:23.393+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-03-17T00:00:00+00:00 [queued]>
[2023-06-01T11:21:23.397+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:21:23.399+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2023-06-01T11:21:23.401+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:21:23.450+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-03-17 00:00:00+00:00
[2023-06-01T11:21:23.462+0000] {standard_task_runner.py:55} INFO - Started process 1129 to run task
[2023-06-01T11:21:23.482+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2022-03-17T00:00:00+00:00', '--job-id', '301', '--raw', '--subdir', 'DAGS_FOLDER/postgres.py', '--cfg-path', '/tmp/tmpi7qi9hrs']
[2023-06-01T11:21:23.486+0000] {standard_task_runner.py:83} INFO - Job 301: Subtask insert_into_table
[2023-06-01T11:21:23.647+0000] {task_command.py:389} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-03-17T00:00:00+00:00 [running]> on host 687ba427e841
[2023-06-01T11:21:23.781+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=insert_into_table
AIRFLOW_CTX_EXECUTION_DATE=2022-03-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-17T00:00:00+00:00
[2023-06-01T11:21:23.784+0000] {sql.py:253} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-03-17', 'dag_with_postgres_operator_v03')
        
[2023-06-01T11:21:23.802+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:21:24.340+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:21:24.360+0000] {sql.py:364} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-03-17', 'dag_with_postgres_operator_v03')
        , parameters: None
[2023-06-01T11:21:24.373+0000] {sql.py:373} INFO - Rows affected: 1
[2023-06-01T11:21:24.394+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20220317T000000, start_date=20230601T112123, end_date=20230601T112124
[2023-06-01T11:21:24.462+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-01T11:21:24.506+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
